judge/views/problem.py

```diff
@@ -309,6 +309,12 @@ def _solved_sort_order(problem):
 
                     queryset = list(queryset)
                     queryset.sort(key=_solved_sort_order, reverse=self.order.startswith('-'))
+            elif sort_key == 'editorial':
+                def _editorial_sort_order(problem):
+                    return int(problem.has_public_editorial())
```

Comment:
This will do O(# of problems) database queries. I would recommend doing something similar to https://github.com/DMOJ/online-judge/blob/master/judge/views/contests.py#L240 and doing the ordering with `order_by`. Also, please add a secondary order so #1445 doesn't happen.

Also, I'm not too sure the usefulness of allowing sorting by a boolean when we also add a toggle.

Reply:
I wasn't sure about this either, but one can sort by solved problems. What do we want to do?

Reply:
Removed sorting entirely.